terraform.tfvars.example

```diff
@@ -179,7 +179,7 @@ image_signing_keys = []
 check_node_active = "all"
 enable_pv_encryption_in_transit = false
 node_pools = {
-  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, label = { app = "frontend", pool = "np1" } }
+  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, label = { app = "frontend", pool = "np1" }, freeform_tags = { app = "frontend", pool = "np1" }, defined_tags = { app = "frontend", pool = "np1" } }
```

Comment:
Instead of using the same k-v pairs as the node labels, perhaps it would be more useful to have different values for freeform and defined tags.